tools/cr/brockit.py

```diff
@@ -977,6 +980,32 @@ def create_push_request(self, issue_url: str) -> bool:
             )
             return False
 
+        if is_uplift:
+            # Only uplift branches set milestones.
+            results = json.loads(
+                terminal.run([
```

Comment:
Eventually (not now!) it would be great to import the GitHub module (https://github.com/brave/brave-core/blob/master/script/lib/github.py) and use / extend that. If anything useful is in the uplift script or other scripts we could add to github.py and re-use here 😄 

Just want to plant a 🐛 in your 👂 (not a bug in the software LOL)

Reply:
I have thought about doing that already yes. There's also the question that I'm using git on the command line too, and I've been told a library could be better. I've been weighing this on my mind for a while now.